### DISCUSSION TITLE
Hey, whats "Prefix"?

### DISCUSSION THREAD

**Discussion by Dexteriority** [winstxnhdw/lc-hax#408] [Q&A] [answered]
Is it function as a panic button?
The hax went off after i bind it to something xD

**Comment by winstxnhdw** [comment, marked as answer]
It just allows you to change `/` commands to whatever character you want.

**Reply by Dexteriority** [reply]
So its basically a bind changer?

**Reply by winstxnhdw** [reply]
Only for commands.

**Reply by Dexteriority** [reply]
I see, thanks!
